packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/index.tsx

```diff
@@ -228,7 +228,6 @@ const AccountRecoveryResetPassword = ({
           // no-op, don't run default
           break;
         default:
```

Comment:
Did we mean to remove that comment? If so is there a point having an epmty deafult block?

Reply:
Yes, the comment was intentionally removed. Much of the `persistVerificationData`/`unpersistVerificationData` logic will no longer needed as we're moving away from email confirmation links (and verification in separate tab). (per @LZoog)

Good call on default not having a purpose here.

Reply:
All reset PW components should be refactored soon enough FWIW. For that `verificationInfo` stuff, I have a note [here](https://docs.google.com/document/d/1K4AD69QgfOCZwFLp7rUcMOkOTslbLCh7jjSdR9zpAkk/edit#heading=h.kkt4eylho93t) that should probably be moved elsewhere - TLDR, we used to store the `context` param in localStorage so if a user verified their account (back when we used links) or reset their PW (with links), the relier/auth_broker was the same as the one they started with. We did this for some specific feature we wanted to persist that we no longer support and we should be using codes all around soon enough.